assets/i18n/en.json

```diff
@@ -519,16 +519,41 @@
     "I18N_SIGNUP_WHY_LICENSE": "Why CC-BY-SA?",
     "I18N_SOLICIT_ANSWER_DETAILS_FEEDBACK": "Okay, now let's go back to your answer.",
     "I18N_SOLICIT_ANSWER_DETAILS_QUESTION": "Could you explain why you picked this answer?",
+    "I18N_SPLASH_BENEFITS_ONE": "Personalized Learning",
+    "I18N_SPLASH_BENEFITS_THREE": "Easy-to-follow Lessons",
+    "I18N_SPLASH_BENEFITS_TITLE": "Our Benefits",
+    "I18N_SPLASH_BENEFITS_TWO": "Story-Based Lessons",
     "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Oppia's lessons, also known as explorations, provide more immersive experiences than static videos or text, helping users learn by doing.",
+    "I18N_SPLASH_FOR_STUDENTS": "For Students",
+    "I18N_SPLASH_FOR_TEACHERS": "For Teachers",
+    "I18N_SPLASH_FOR_VOLUNTEERS": "For Volunteers",
+    "I18N_SPLASH_ICON_ONE_TEXT": "1 Million+ Users",
+    "I18N_SPLASH_ICON_THREE_TEXT": "40 Curated Lessons",
+    "I18N_SPLASH_ICON_TWO_TEXT": "Available in 4+ Languages",
     "I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href=\"<[hrefUrl]>\">click here.</a>",
     "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "Thank you.",
     "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "We Need JavaScript in Your Browser",
+    "I18N_SPLASH_LEARN_MORE": "Learn More",
     "I18N_SPLASH_PAGE_TITLE": "Oppia | Free, Online and Interactive Lessons for Anyone",
     "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "Explorations are simple to create. They are easily adapted based on individual student feedback and trends in the experience of your learners around the world.",
     "I18N_SPLASH_SITE_FEEDBACK": "Site Feedback",
-    "I18N_SPLASH_SUBTITLE": "Oppia makes it easy to create interactive lessons that educate and engage.",
+    "I18N_SPLASH_START_CONTRIBUTING": "Start Contributing",
+    "I18N_SPLASH_STUDENTS_CONTENT": "Oppia identifies common wrong answers and provides tailored feedback, so that students get a personalized experience. Our lessons retain engagement through playful characters that lead students and diverse strategies that solidify knowledge. Check out our math lessons with proven results!",
```

Comment:
Change second sentence to "Our lessons keep students engaged through playful characters, and use different strategies to solidify their knowledge." This probably needs further work in the future but at least it's less jargon-y.

Reply:
Done